tools/ide_plugins/pysa-vscode/package.json

```diff
@@ -38,5 +38,34 @@
     "tslint-microsoft-contrib": "^5.0.1",
     "typescript": "^3.5.1",
     "vscode": "^1.1.7"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "contributes": {
+      "languages": [{
+          "id": "pysa",
+          "aliases": ["Pysa", "pysa"],
+          "extensions": [".pysa"],
+          "configuration": "./language-configuration.json"
+      },
+      {
+          "id": "taint",
+          "aliases": ["Pysa", "taint"],
+          "extensions": ["taint.config"],
+          "configuration": "./language-configuration.json"
+      }
+      ],
+      "grammars": [{
```

Comment:
Formatting is pretty inconsistent within that file. In my opinion, we should break a line here and indent the following object. Could we use an auto formatter here?

Reply:
Thanks for catching that, I just used Prettier to fix it - let me know if it looks better now!